src/Atoms/HomeElements/DarkMode.test.js

```diff
@@ -0,0 +1,17 @@
+import { render, screen } from '@testing-library/react';
+
+import { MyDarkMode } from './DarKMode';
+
+describe('Dark Mode Button', () => {
+  test('should contains a button', () => {
+    const result = render(MyDarkMode());
+    const headerElement = result.container.querySelector('button:first-of-type');

```

Comment:
For a button, you can use `getByRole`, see: https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-query-variants-for-anything-except-checking-for-non-existence

Reply:
Thank you so much, I had some dificulties to get that result.